wdl-format/tests/format/ENCODE-DCC_chip-seq-pipeline/source.wdl

```diff
@@ -0,0 +1,3296 @@
+version 1.0
```

Comment:
Just a sidenote—before we pull _any_ external scripts in, we need to make sure the license is compatible with ours. So, for any that you add here that weren't written by our team, please add a `LICENSE.txt` that describes the license at the time we retrieved it to the folder and also add a comment to the top saying (a) when it was retrieved, (b) what commit link it was retrieved at and (c) what license it was pulled in under.

Reply:
Got it! 💯